docs/data/data-grid/recipes-editing/SingleClickEditing.tsx

```diff
@@ -80,7 +80,10 @@ export default function SingleClickEditing() {
       }
 
       // Ignore portal
-      if (!event.currentTarget.contains(event.target as Element)) {
+      if (
+        (event.target as any).nodeType === 1 &&
+        !event.currentTarget.contains(event.target as Element)
+      ) {
```

Comment:
can't we use the `isEventTargetInPortal` here as well?
```suggestion
      if (isEventTargetInPortal(event)) {
```

Reply:
or can't we because its coming from internals?

Reply:
Yeah, I didn't export it publicly and also wanted to keep it explicit in the recipe